catkin_home/src/navigation/map_visualizer/scripts/goal_display.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+import roslib; roslib.load_manifest('visualization_marker_tutorials')
```

Comment:
Add a file description and functions description to this and also the other python files.